nemo/collections/multimodal/data/energon/task_encoder.py

```diff
@@ -54,7 +61,15 @@ class MultiModalTaskEncoder(
     for model input.
     """
 
-    def __init__(self, tokenizer, image_processor, multimodal_sample_config):
+    def __init__(
+        self,
+        tokenizer,
+        image_processor,
+        multimodal_sample_config,
+        packed_sequence=False,
+        packing_seq_length=4096,
```

Comment:
can you use consistent naming as fine_tuning.py? i.e. 
use one variable `packed_sequence_size`, and `packed_sequence_size=-1` indicates not using packed sequence

Reply:
i kept `packed_sequence` here for consistency with other vlm data modules where we don't use packed_sequence_size